includes/User.php

```diff
@@ -2702,6 +2708,25 @@ public function setLocalAttribute($attribute, $value, $cityId = null, $sep = '-'
 	 */
 	public function setGlobalAttribute($attribute, $value) {
 		$this->setOptionHelper($attribute, $value);
+		$this->setAttributeInService($attribute, $value);
+	}
+
+	/**
+	 * Sets an attribute into the User Attribute Service
+	 *
+	 * @param $attributeName
+	 * @param $attributeValue
+	 */
+	private function setAttributeInService($attributeName, $attributeValue) {
```

Comment:
Normally I try and be a zealot when it comes to whitespace in PHP Land, but since this is core code which doesn't follow that convention (and I didn't want to run the stylize script on this 5k line file), I figured it would be better to be consistent here :)